README.md

```diff
@@ -1,19 +1,23 @@
 # Rare: The Puplishing Platform for the Discerning Doggo
 
-This is a learning project. Our goals were to practice full stack development by creating a react client and python-sql back end.
+This is a learning project. Our goals were to practice full stack development by creating a react client and python-django back end.
 
-This project was completed in a 6 day Agile-SCRUM sprint. At the start of the project we met with our project manager and assigned point scales to the enhancement tickets based on our estimates of the workload.
+This project was completed in 3 agile-scrum sprints. 
+- Week one, wrote the back end in pure python + sql. That repo can be found here: https://github.com/NSS-Day-Cohort-42/rare-server-news-hounds
+- Week two, re-wrote the back end using the django framework: 
+https://github.com/NSS-Day-Cohort-42/news_hounds_django_server
+- Week three, added more functionality to the site (subscriptions & reactions)
 
 As a team, we decided to implement the react-bootstrap styling library on this project. This allowed us to create a consistent appearance across the site.
 
 ## Installation Instructions 
 Requirements:
-NPM (https://github.com/NSS-Day-Cohort-42/rare-server-news-hounds)
+NPM (https://www.npmjs.com/)
 
 1. In the terminal, run `git clone https://github.com/NSS-Day-Cohort-42/rare-news-hounds.git`
```

Comment:
If we want to be super duper hyper ultra mega explicit (like the server README is) we could add a step after cloning to `cd rare-news-hounds`. I'm approving regardless but maybe something to add if you want to.